.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
Please be sure to remove the test.md file from your remote repository 🙌 Well done applying all required changes by the previous reviewer!